jeremy-aaron-tre/vendor/styles/modules.css

```diff
@@ -0,0 +1,3 @@
+.template {
+  display: none;
+}
```

Comment:
your regular CSS files should go in a `styles` folder that is at the same level as your `scripts` folder. the `vendor` folder would be for things such as normalize, reset, or icon CSS files